src/phy.c

```diff
@@ -338,3 +338,176 @@ static int handle_ed_scan(struct nl802154_state *state,
 
 COMMAND(get, ed_scan, "[<page> [<channels> [<duration>]]]",
     NL802154_CMD_ED_SCAN_REQ, 0, CIB_PHY, handle_ed_scan, NULL);
+
+
+static int print_nl802154_beacon_notify_ind(struct nl_msg *msg, void *arg)
```

Comment:
handle_beacon_notify() is already the iwpan command handler.  I have been using the print_xxx for the callbacks that print received data to the terminal.  eg. print_ed_scan_handler().